public/distinctions.json

```
@@ -0,0 +1,23 @@
+{
+  "Paris Peace Forum": {
+    "description": {
+      "fr": "Sélectionné comme solution par le Forum de Paris sur la paix 2020",
+      "en": "Selected as Solution by the Paris Peace Forum 2020"
+    },
+    "link": "https://parispeaceforum.org/projects/community-against-disinfo/"
+  },
+  "Nobel Prize Summit": {
+    "description": {
+      "fr": "Présenté comme une «innovation de source ouverte pour l'intégrité de l'information» au sommet du prix Nobel 2023",
```

Non-breakable space inside quotes in French 🙂 
```suggestion
      "fr": "Distingué comme « Innovation ouverte pour l'intégrité de l'information » au Sommet du prix Nobel 2023",
```